### ISSUE TITLE
Import just mixins, no generated classes?

### ISSUE THREAD

**Issue by Jaspur** [IanLunn/Hover#37]
Is there an easy way to include just the mixins? When I include the hover.scss from bower, it also includes the generated classes for the hover-effect.

**Comment by IanLunn** [comment]
I changed my mind on adding this file as I don't think that many people will use it and it means another file to maintain. Is there a problem with using the predefined Hover classes? #4 has requested prefixed class names to reduce the chance of conflicts which will be addressed in the upcoming 2.0.

If you or anyone else wants to go ahead with this idea, the `hover-mixins.scss` file would look something like this:

```css
@import "options";
@import "mixins";
@import "hacks";

@import "effects/grow";
@import "effects/shrink";
// Repeat for the other effects
```

Then you can add an effect's styles to your own class, like so:

```css
.myclass {
    @include grow();
}
```

**Comment by Jaspur** [comment]
I don't use classes, only the @include for adding an effect. When including ```scss/hover.scss```now, and I wanna use two effects, I get all the classes, and is the css-file bigger then it has to be. 
I'll make a PR, then you can decide what you wanna do with it ;-)
